website/static/js/pages/profile-settings-addons-page.js

```diff
@@ -1,7 +1,15 @@
+'use strict';
+
+require('css/user-addon-settings.css');
 var $ = require('jquery');
+var ko = require('knockout');
 var bootbox = require('bootbox');
-var osfHelpers = require('osfHelpers');
+var Raven = require('raven-js');
+var $osf = require('js/osfHelpers');
```

Comment:
Separate 3rd party libs and internal module with a blank line: http://cosdev.readthedocs.org/en/latest/style_guides/javascript.html#commonjs-modules